pkg/jvscrypto/rotation_handler.go

```diff
@@ -221,34 +221,42 @@ func (h *RotationHandler) shouldDisable(ctx context.Context, ver *kmspb.CryptoKe
 	if shouldDisable {
 		logger.Info("version created before cutoff date, should disable.", zap.Any("version", ver), zap.Any("cutoff", cutoff))
 	} else {
-		logger.Debug("version created before cutoff date, no action necessary.", zap.Any("version", ver), zap.Any("cutoff", cutoff))
+		logger.Debug("version created after cutoff date, no action necessary.", zap.Any("version", ver), zap.Any("cutoff", cutoff))
 	}
 	return shouldDisable
 }
 
-func (h *RotationHandler) shouldRotate(ctx context.Context, ver *kmspb.CryptoKeyVersion, curTime time.Time) bool {
+func (h *RotationHandler) shouldRotate(ctx context.Context, primary *kmspb.CryptoKeyVersion, newest *kmspb.CryptoKeyVersion, curTime time.Time) bool {
```

Comment:
Can we add some documentation to this now about the conditions under which it returns `true` and the conditions under which it returns `false`?

Reply:
Updated.